pydoctor/epydoc2stan.py

```diff
@@ -184,6 +184,11 @@ def _resolve_identifier_xref(self,
             target = src.resolveName(identifier)
             if target is not None:
                 return target
+            if isinstance(src, model.Class):
+                for base in src.allbases():
```

Comment:
Does the `Class.find()` method provide what you need? That would reduce the amount of new code considerably.

Reply:
I don't know to be honest. I have replaced this with `target = src.find(identifier)` and it seems to do the trick (i.e. all references that were resolved with the previous version of the code in `python-igraph`'s documentation are also resolved with the new version), but I don't know what the difference is. if you think it's better to use `Class.find()`, I trust your assessment and I'll just use that here.